books/free-programming-books-si.md

```diff
@@ -0,0 +1,10 @@
+### Index
+
+* [C++](#cpp)
+
+
+### <a id="cpp"></a>C++
+
+* [C++ පරිගණක ක්‍රමලේඛනය මුල  සිට  සරලව](http://www.mediafire.com/file/229c1v9n86bw2yf/C-Sinhala_Lanka_Learning.pdf/file) - කසුන් ලියනගේ 
```

Comment:
Please provide official links to the book

Reply:
The book is available on a local website for downloading (https://www.aluth.com/)
web post link: https://www.aluth.com/2018/06/c-learning-in-sinhala-pdf-book.html

Reply:
the download button is in sinhala language as  [මෙතනින් ] in the last line of the paragraph "එසේනම් [මෙතනින් ]පිවිස මෙම පොත බාගත කරගන්න."